### DISCUSSION TITLE
What is the correct use of `priority` ?

### DISCUSSION THREAD

**Discussion by tmlmt** [creativeprojects/resticprofile#389] [Q&A]
1. The [docs] gives a table with capitalized string values, and an example with a lower case "low". 
When I try with a Capitalized "Low", resticprofile complains that an incorrect value is given for the `priority` parameter

2. The table in the docs says that "low" corresponds to a Nice value of `10`, but with the following global section: 
```yaml
global:
  priority: low
```
Executing `resticprofile schedule --all` creates systemd files which contain `Nice=5`
Is the niceness of the systemd unit different from the niceness of the restic command executed by the systemd unit?

Could anyone clarify?

**Comment by creativeprojects** [comment]
1. It looks like we only accept values in lowercase. I'll update the code to accept any case, it's too restrictive indeed.
2. You're right: it is confusing: when running a systemd timer, the Nice value is fixed at `5`. There's already an issue opened about it https://github.com/creativeprojects/resticprofile/issues/229 😞 

I'm making a note to update the documentation, thank you for your patience 😉

**Reply by tmlmt** [reply]
Thanks for clarifying, @creativeprojects. Well noted. And thanks for resticprofiles, it’s been a great replacement to the previous solution I was using to schedule restic backups 👍
